pkg/operator/provisioner.go

```diff
@@ -83,22 +85,16 @@ func (p *rookVolumeProvisioner) Provision(options controller.VolumeOptions) (*v1
 
 	imageName := createImageName(options.PVName)
 
-	rookClient, err := p.clusterManager.getRookClient(p.provConfig.clusterNamespace)
-	if err != nil {
-		return nil, fmt.Errorf("Failed to get rook client: %v", err)
-	}
-
-	res, err := createVolume(imageName, p.provConfig.pool, requestBytes, rookClient)
+	err = p.createVolume(imageName, p.provConfig.pool, requestBytes)
 	if err != nil {
 		return nil, err
 	}
-	logger.Infof("Rook block image created: %s", res)
 
-	rookClientInfo, err := rookClient.GetClientAccessInfo()
+	monitors, err := p.getMonitorEndpoints()
 	if err != nil {
-		return nil, fmt.Errorf("Failed to get rook client information: %v", err)
+		return nil, fmt.Errorf("Failed to get mon endpoints: %v", err)
```

Comment:
all the errors coming back from this function already have "failed to get mon endpoints" in their message.  no need to repeat it here.